pkg/csi/service/vanilla/controller_helper.go

```diff
@@ -139,3 +144,52 @@ func convertCnsVolumeType(ctx context.Context, cnsVolumeType string) string {
 	}
 	return volumeType
 }
+
+func (c *controller) getBlockVolumeToHostMap(ctx context.Context,
```

Comment:
Take `c.Manager` as input to this function to avoid dependency on `controller` object which is very big object.